<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>906</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(68, 45, 34);
color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Student Management System</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="registration">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>45</height>
                </size>
               </property>
               <property name="text">
                <string>REGISTRATION</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="search">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>SEARCH</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="showall">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>SHOW ALL RECORD</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="update">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>45</height>
                </size>
               </property>
               <property name="text">
                <string>UPDATE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="delete_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>DELETE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="logout">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>45</height>
                </size>
               </property>
               <property name="text">
                <string>LOGOUT</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QFrame" name="frame_5">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QFrame" name="frame_7">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <property name="leftMargin">
                    <number>40</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="minimumSize">
                      <size>
                       <width>50</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
                     </property>
                     <property name="text">
                      <string>REG NO</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="regno">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="placeholderText">
                      <string>REG NO</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="check2">
                     <property name="minimumSize">
                      <size>
                       <width>75</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>CHECK </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="check">
                     <property name="minimumSize">
                      <size>
                       <width>75</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Check</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_8">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QFrame" name="frame_11">
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_7">
                      <property name="leftMargin">
                       <number>30</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="label_3">
                        <property name="minimumSize">
                         <size>
                          <width>370</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
                        </property>
                        <property name="text">
                         <string>FIRST NAME</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="firstname">
                        <property name="minimumSize">
                         <size>
                          <width>360</width>
                          <height>40</height>
                         </size>
                        </property>
                        <property name="placeholderText">
                         <string>FIRST NAME</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_9">
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_8">
                      <property name="leftMargin">
                       <number>30</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="label_4">
                        <property name="minimumSize">
                         <size>
                          <width>370</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
                        </property>
                        <property name="text">
                         <string>LAST NAME</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="lastname">
                        <property name="minimumSize">
                         <size>
                          <width>360</width>
                          <height>40</height>
                         </size>
                        </property>
                        <property name="placeholderText">
                         <string>LAST NAME</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_10">
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_9">
                      <property name="leftMargin">
                       <number>30</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="label_5">
                        <property name="minimumSize">
                         <size>
                          <width>370</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
                        </property>
                        <property name="text">
                         <string>GENDER</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="gender">
                        <property name="minimumSize">
                         <size>
                          <width>360</width>
                          <height>40</height>
                         </size>
                        </property>
                        <property name="placeholderText">
                         <string>GENDER</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_12">
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_10">
                      <property name="leftMargin">
                       <number>30</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="label_6">
                        <property name="minimumSize">
                         <size>
                          <width>370</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
                        </property>
                        <property name="text">
                         <string>DOB</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="dob">
                        <property name="minimumSize">
                         <size>
                          <width>360</width>
                          <height>40</height>
                         </size>
                        </property>
                        <property name="placeholderText">
                         <string>0/0/0000</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_14">
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_11">
                      <property name="leftMargin">
                       <number>30</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="label_7">
                        <property name="minimumSize">
                         <size>
                          <width>370</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
                        </property>
                        <property name="text">
                         <string>COURSE</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="course">
                        <property name="minimumSize">
                         <size>
                          <width>360</width>
                          <height>40</height>
                         </size>
                        </property>
                        <property name="placeholderText">
                         <string>COURSE</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_13">
                     <property name="styleSheet">
                      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <item>
                       <widget class="QPushButton" name="clear_btn">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>35</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>CLEAR</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="updt">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>35</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>UPDATE RECORD</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="add">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>35</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>ADD RECORD</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="delete_btn">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>35</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>DELETE RECORD</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="display">
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
